src/style.css

```diff
@@ -0,0 +1,30 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  width: 100%;
```

Comment:
your stars container is a div, is display is block, so the width is 100% by default